test-cases/gutenberg/spacer.md

```diff
@@ -0,0 +1,69 @@
+# Spacer Block - Test Cases
+
+--------------------------------------------------------------------------------
+
+##### TC001
+
+### Spacer is rendered without crash
+
+-   Add a `Spacer` block
+-   Check if `Spacer` is rendered
+-   Check if app do not crash when open settings
+
+Expected look:  
+![Spacer](../resources/spacer.png)
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+### Settings: Adjusting spacer height with slider
+
+-   Add a `Spacer` block
+-   Check if `Spacer` block is rendered with default height of `100px`
+-   Change slider value under `Spacer` settings and measure if `Spacer` block has height setted via slider
+
+Expected look:  
+![SpacerHeight](../resources/spacer-height.png)
+
+--------------------------------------------------------------------------------
+
+##### TC003
+
+### Settings: Check range of values
+
+-   Add a `Spacer` block
+-   Check if available `Spacer` height range to set via slider in options is between `20px` and `500px`
+
+Expected look:  
+![SpacerRanges](../resources/spacer-max-height.png)
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Settings: Check if height range extends when Spacer was saved in web version
+
+-   Add a `Spacer` block via web version
+-   Set `Spacer` height to more that `500px` and save the post
+-   Save the post
+-   Open mobile app
+-   Check if `Spacer` block is rendered with setted height
+-   Check if slider value in options is in the middle of available range
+-   The max value should be twice as setted on web
+-   The min value should be unchanged and equals `20px`
+
+Expected look:  
+![SpacerExceedHeight](../resources/spacer-exceed-height.png)
+
+--------------------------------------------------------------------------------
+
+##### TC005
+
+### Spacer in horizontal layout works as expected
+
+-   Add a `Spacer` block
+-   Rotate the device to horizontal layout
+-   Check if `Spacer` has proper height according to settings
+
+--------------------------------------------------------------------------------
```

Comment:
More comments:

* I would like to have test case similar to TC002, but the value is set using input.
* If you will change title remember to change them in `test-suites/gutenberg/sanity-tests.md` as well

Reply:
I have added TC for manipulate height with textinput and update titles in sanity-test.md